benchmarks/osb/README.md

```diff
@@ -106,26 +106,28 @@ use an algorithm that requires training.
 
 #### Parameters
 
-| Name                                    | Description                                                                      |
```

Comment:
why there are changes in the readme? Are these the results which has changed?

Reply:
When I update procedures, I update README output examples to make sure I dont miss any metrics that change.